readme.md

```diff
@@ -64,13 +64,25 @@ While it is *technically* possible to setup EasyOpenCV for OnBotJava, it is much
          
 6. Open the `build.common.gradle` file, and find the line `minSdkVersion 19`, and replace it with `minSdkVersion 23`
 
-7. Now perform a Gradle Sync:
+7. In the same file, `build.common.gradle`, scroll down until you find this part:
+
+    <img src="doc/images/build-common-gradle-buildscript.png" width="50%" height="50%">
+
+8. Remove both references to `"arm64-v8a"` described in debug and release blocks, leaving `"armeabi-v7a"` untouched:
+
+    ![img-here](doc/images/build-common-gradle-buildscript-remove-arm64.png)
+
+\**If you already had arm64 removed no further changes are needed*
+
+\*\**This step is needed since the OpenCV native library is 32-bits, and by modifying this file will make the application run in 32-bit mode. Otherwise, EasyOpenCV won't be able to load the libary and a warning will be shown when opening the app indicating that OpenCV-enabled OpModes will crash.*
```

Comment:
Wording is a bit strange on the first sentence here. Maybe something like *"This step is needed because the OpenCV native library is compiled for 32-bit, and Android applications cannot load both 32-bit and 64-bit libraries at the same time."*